### ISSUE TITLE
[NTR] Impaired fasting glucose & some questions

### ISSUE THREAD

**Issue by mshadbolt** [obophenotype/human-phenotype-ontology#7207]
Hi, I am hoping to get a few terms related to Diabetes added. I think that `Impaired fasting glucose` is a fairly straight forward one. The other categories I am trying to ontologise from a Diabetes study are `previously diagnosed diabetes`, [`newly diagnosed diabetes`](https://browser.ihtsdotools.org/?perspective=full&conceptId1=405749004&edition=MAIN/2021-07-31&release=&languages=en) & [`normal glucose tolerance`](https://browser.ihtsdotools.org/?perspective=full&conceptId1=166926006&edition=MAIN/2021-07-31&release=&languages=en). Do you think these are within the scope of HPO and worth adding? Some have entries within SNOMED but wasn't sure if the lack of a phenotype or a 'new' phenotype is an appropriate thing to add to HPO. If you think any would be worth adding I can make new term request tickets. OR also if you know of another ontology where these terms might fit better let me know. Thanks!!

**Preferred term label:**
Impaired fasting glucose

(This was mentioned in this issue #3377 but I don't believe it has been added as a specific term. It is also mentioned in the definition of the proposed parent - Glucose intolerance.)

**Synonyms**
Impaired fasting glycaemia, Impaired fasting glycemia, IFG

**Definition (free text, please give PubMed ID)**

Impaired fasting glucose (IFG) is indicated by a fasting plasma glucose above normal but below the diabetic range. Levels between 110 mg/dl (6.1 mmol/l) to 125 mg/dl (6.9 mmol/l) are diagnostic of IFG.

PMID: 17327355
PMID: 29763085

cross ref to SNOMED: http://snomed.info/id/390951007

**Parent term (use hpo.jax.org/app)**

[Glucose intolerance HP:0001952](https://hpo.jax.org/app/browse/term/HP:0001952)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
I believe this one: https://www.omim.org/entry/125853 but it is a bit confusing because there are so many diabetes entries.

**Your nano-attribution (ORCID)**
0000-0001-7505-5418

**Comment by pnrobinson** [comment]
Sorry for the delay in adding this term, which I just did.
The way of expressing Normal glucose tolerance would be by negative the term [Glucose intolerance HP:0001952](https://hpo.jax.org/app/browse/term/HP:0001952). There are many ways of doing this but in principle somewhere in your database you would have a column for "NOT" or something like that. We have recently completed the GA4GH [phenopacket schema](https://www.medrxiv.org/content/10.1101/2021.11.27.21266944v1.full) that is specifically intended for things like this

**Comment by pnrobinson** [comment]
The concepts newly diagnosed and previously diagnosed diabetes are out of scope for HPO. Again the phenopacket schema would provide one way of doing this, but I would suggest just adding some additional field in whatever database you are using for the study would be the simplest way.
